### ISSUE TITLE
Add support for positional arguments to lncli

### ISSUE THREAD

**Issue by Roasbeef** [lightningnetwork/lnd#31]
Currently, `lncli` (`cmd/lncli`) only support fully specifying each argument for each command. This parsing should be extended to also allow positional arguments, and also combo of positional+keyword arguments.

**Comment by tompre** [comment]
I would take this one.
I also have some ideas for bash completion for pubkey or channel ids.
Whats your take on that?

**Comment by Roasbeef** [comment]
@tompre excellent! 

In order to do bash completion for the pubkey's channelID's, you'll need to before to a call to `describegraph` in order to obtain the available list of pubkey's/chanID's. If we had a sort of "console" mode, you could put both sets in their own trie then use that for auto-completion.

**Comment by tompre** [comment]
I wanted to use the bash completion functionality provided by urfave/cli. [link](https://github.com/urfave/cli#bash-completion)
Basic completion can be enabled by a single flag (`app.EnableBashCompletion = true`) and completion candidates are generated in go using the `BashComplete` flag in the command definition.
Unfortunately @aantonop did not see this before writing pull request #109 and I did not see his pull request before it was merged.
